src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-configuration-form/rbd-configuration-form.component.html

```diff
@@ -6,7 +6,8 @@
        class="col-12">
     <h3 class="cd-header">
       <span (click)="toggleSectionVisibility(section.class)"
-        class="collapsible">{{ section.heading }} <i [ngClass]="!sectionVisibility[section.class] ? icons.addCircle : icons.minusCircle" aria-hidden="true"></i></span>
+        class="collapsible">{{ section.heading }} <i [ngClass]="!sectionVisibility[section.class] ? icons.addCircle : icons.minusCircle"
```

Comment:
Is the indentation correct here?

Reply:
Not really, fixed it.
Went back to test the other plugins and seems none of them do a good job validating the indentation of attributes.

Reply:
> Went back to test the other plugins and seems none of them do a good job validating the indentation of attributes.

Meh :/ But thank you!